go.mod

```diff
@@ -1,17 +1,12 @@
-module BuxOrg/bux-admin-key-generator
+module github.com/bitcoin-sv/spv-wallet-admin-keygen
 
 go 1.21
 
 toolchain go1.21.4
 
+require github.com/BuxOrg/go-buxclient v0.4.0
```

Comment:
We will rename this after go-buxclient renaming ?

Reply:
![image](https://github.com/BuxOrg/bux-admin-key-generator/assets/48433067/059bd99a-991e-4eec-8779-7cc3218e146f)

Yes, exactly